locales/app.yml

```
@@ -1208,30 +1208,14 @@ _version: 2
   zh_CN: "Windows 更新"
   zh_TW: "Windows 更新"
   de: "Windows-Update"
-"Would check if OpenBSD is -current":
-  en: "Would check if OpenBSD is -current"
-  lt: "Patikrintų, ar OpenBSD yra -current"
-  es: "Comprobaría si OpenBSD está en -current"
-  fr: "Vérifierait si OpenBSD est à -curent"
-  zh_CN: "将检查 OpenBSD 是否为 -current"
-  zh_TW: "會檢查 OpenBSD 是否為 -current"
-  de: "Würde überprüfen, ob OpenBSD -current ist"
-"Would upgrade the OpenBSD system":
-  en: "Would upgrade the OpenBSD system"
-  lt: "Atnaujintų OpenBSD sistemą"
-  es: "Actualizaría el sistema OpenBSD"
-  fr: "Mettrait à jour le système OpenBSD"
-  zh_CN: "将升级 OpenBSD 系统"
-  zh_TW: "會升級 OpenBSD 系統"
-  de: "Würde das OpenBSD-System aktualisieren"
-"Would upgrade OpenBSD packages":
-  en: "Would upgrade OpenBSD packages"
-  lt: "Atnaujintų OpenBSD paketus"
-  es: "Actualizaría los paquetes de OpenBSD"
-  fr: "Mettrait à jour les paquets OpenBSD"
-  zh_CN: "将升级 OpenBSD 软件包"
-  zh_TW: "會升級 OpenBSD 套件"
-  de: "Würde OpenBSD-Pakete aktualisieren"
+"Checking if /etc/motd contains -current or -beta":
+  en: "Checking if /etc/motd contains -current or -beta"
+  lt: "Tikrinimas, jei /etc/motd yra -srovė arba -beta"
+  es: "Comprobación de si /etc/motd contiene -current o -beta"
+  fr: "Vérification si /etc/ Motd contient -current ou -beta"
+  zh_CN: "检查 /etc/motd 是否包含 -current 或 -beta"
+  zh_TW: "检查 /etc/motd 是否包含 -current 或 -beta"
+  de: "Überprüfen Sie, ob /etc/motd -current oder -beta enthält"
```

This is phrased as a question ("(please) check if /etc/motd contains -current or -beta"), note "Sie" (meaning "you")
Better would be: "Überprüfen, ob /etc/motd -current oder -beta enthält"
```suggestion
  de: "Überprüfen, ob /etc/motd -current oder -beta enthält"
```

---

Google translate strikes again